prowler/providers/common/provider.py

```diff
@@ -0,0 +1,29 @@
+from abc import ABC, abstractmethod
+
+
+class CloudProvider(ABC):
+    audit_resources: list = None
+    is_quiet: bool
+    output_modes: list
+    output_directory: str
+    allowlist_file: str
+    bulk_checks_metadata: dict
+    verbose: str
+    output_filename: str
+    only_logs: bool
+    unix_timestamp: bool
```

Comment:
Why there are output options stored in the `CloudProvider`?

Reply:
Deleted, at the beginning they were included thinking about including the outputs changes in this PR but maybe is better to include it in the future